### ISSUE TITLE
Issue installing latest from Debian download site: sip-communicator.properties: No such file or directory

### ISSUE THREAD

**Issue by robertoandrade** [jitsi/jitsi-meet#340]
I've been running automated installs of the latest JVB/meet from the Debian repo and just recently started receiving the following error when attempting to install `jitsi-meet` via `apt-get`:

```
root@19e5dc1063ce:/# echo 'deb http://download.jitsi.org/nightly/deb unstable/' >> /etc/apt/sources.list

root@19e5dc1063ce:/# wget -qO - https://download.jitsi.org/nightly/deb/unstable/archive.key | apt-key add -
OK

root@19e5dc1063ce:/# apt-get update
Hit http://security.debian.org jessie/updates InRelease
Ign http://download.jitsi.org unstable/ InRelease
Hit http://http.debian.net jessie-backports InRelease
Get:1 http://security.debian.org jessie/updates/main amd64 Packages [160 kB]
Hit http://httpredir.debian.org jessie InRelease
Get:2 http://download.jitsi.org unstable/ Release.gpg [72 B]
Hit http://httpredir.debian.org jessie-updates InRelease
Get:3 http://download.jitsi.org unstable/ Release [1547 B]
Get:4 http://http.debian.net jessie-backports/main amd64 Packages [232 kB]
Get:5 http://download.jitsi.org unstable/ Packages [25.6 kB]
Get:6 http://httpredir.debian.org jessie/main amd64 Packages [9038 kB]
Get:7 http://httpredir.debian.org jessie-updates/main amd64 Packages [3616 B]
Fetched 9461 kB in 8s (1139 kB/s)
Reading package lists... Done

root@19e5dc1063ce:/# apt-get -y install jitsi-meet
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following extra packages will be installed:
  authbind default-jre-headless javascript-common jicofo jitsi-meet-prosody jitsi-videobridge libjs-jquery libjs-jquery-ui libjs-strophe openjdk-7-jre-headless tzdata tzdata-java
Suggested packages:
  default-jre apache2 lighttpd httpd libjs-jquery-ui-docs icedtea-7-jre-jamvm libnss-mdns sun-java6-fonts fonts-ipafont-gothic fonts-ipafont-mincho ttf-wqy-microhei ttf-wqy-zenhei fonts-indic
The following NEW packages will be installed:
  authbind default-jre-headless javascript-common jicofo jitsi-meet jitsi-meet-prosody jitsi-videobridge libjs-jquery libjs-jquery-ui libjs-strophe openjdk-7-jre-headless tzdata-java
The following packages will be upgraded:
  tzdata
1 upgraded, 12 newly installed, 0 to remove and 0 not upgraded.
Need to get 66.5 MB of archives.
After this operation, 112 MB of additional disk space will be used.
Get:1 http://security.debian.org/ jessie/updates/main openjdk-7-jre-headless amd64 7u79-2.5.6-1~deb8u1 [39.7 MB]
Get:2 http://download.jitsi.org/nightly/deb/ unstable/ jitsi-videobridge 494-1 [13.9 MB]
Get:3 http://httpredir.debian.org/debian/ jessie-updates/main tzdata all 2015e-0+deb8u1 [192 kB]
Get:4 http://httpredir.debian.org/debian/ jessie-updates/main tzdata-java all 2015e-0+deb8u1 [80.5 kB]
Get:5 http://httpredir.debian.org/debian/ jessie/main default-jre-headless amd64 2:1.7-52 [9072 B]
Get:6 http://httpredir.debian.org/debian/ jessie/main javascript-common all 11 [6120 B]
Get:7 http://httpredir.debian.org/debian/ jessie/main libjs-jquery-ui all 1.10.1+dfsg-1 [499 kB]
Get:8 http://httpredir.debian.org/debian/ jessie/main authbind amd64 2.1.1 [20.2 kB]
Get:9 http://httpredir.debian.org/debian/ jessie/main libjs-strophe all 1.1.3+dfsg-0.1 [59.4 kB]
Get:10 http://download.jitsi.org/nightly/deb/ unstable/ jicofo 1.0-121-1 [11.1 MB]
Get:11 http://download.jitsi.org/nightly/deb/ unstable/ libjs-jquery 2.1.1-1 [94.7 kB]
Get:12 http://download.jitsi.org/nightly/deb/ unstable/ jitsi-meet-prosody 1.0.603-1 [5032 B]
Get:13 http://download.jitsi.org/nightly/deb/ unstable/ jitsi-meet 1.0.603-1 [777 kB]
Fetched 66.5 MB in 1min 38s (674 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
(Reading database ... 18223 files and directories currently installed.)
Preparing to unpack .../tzdata_2015e-0+deb8u1_all.deb ...
Unpacking tzdata (2015e-0+deb8u1) over (2015d-0+deb8u1) ...
Setting up tzdata (2015e-0+deb8u1) ...

Current default time zone: 'Etc/UTC'
Local time is now:      Wed Aug 12 02:32:02 UTC 2015.
Universal Time is now:  Wed Aug 12 02:32:02 UTC 2015.
Run 'dpkg-reconfigure tzdata' if you wish to change it.

Selecting previously unselected package tzdata-java.
(Reading database ... 18223 files and directories currently installed.)
Preparing to unpack .../tzdata-java_2015e-0+deb8u1_all.deb ...
Unpacking tzdata-java (2015e-0+deb8u1) ...
Selecting previously unselected package openjdk-7-jre-headless:amd64.
Preparing to unpack .../openjdk-7-jre-headless_7u79-2.5.6-1~deb8u1_amd64.deb ...
Unpacking openjdk-7-jre-headless:amd64 (7u79-2.5.6-1~deb8u1) ...
Selecting previously unselected package default-jre-headless.
Preparing to unpack .../default-jre-headless_2%3a1.7-52_amd64.deb ...
Unpacking default-jre-headless (2:1.7-52) ...
Selecting previously unselected package jitsi-videobridge.
Preparing to unpack .../jitsi-videobridge_494-1_amd64.deb ...
Unpacking jitsi-videobridge (494-1) ...
Processing triggers for systemd (215-17+deb8u1) ...
Setting up tzdata-java (2015e-0+deb8u1) ...
Setting up openjdk-7-jre-headless:amd64 (7u79-2.5.6-1~deb8u1) ...
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/rmid to provide /usr/bin/rmid (rmid) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/keytool to provide /usr/bin/keytool (keytool) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/pack200 to provide /usr/bin/pack200 (pack200) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/rmiregistry to provide /usr/bin/rmiregistry (rmiregistry) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/unpack200 to provide /usr/bin/unpack200 (unpack200) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/orbd to provide /usr/bin/orbd (orbd) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/servertool to provide /usr/bin/servertool (servertool) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/tnameserv to provide /usr/bin/tnameserv (tnameserv) in auto mode
update-alternatives: using /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/jexec to provide /usr/bin/jexec (jexec) in auto mode
Setting up default-jre-headless (2:1.7-52) ...
Setting up jitsi-videobridge (494-1) ...
useradd: warning: the home directory already exists.
Not copying any file from skel directory into it.
invoke-rc.d: policy-rc.d denied execution of start.
Processing triggers for libc-bin (2.19-18) ...
Processing triggers for systemd (215-17+deb8u1) ...
Selecting previously unselected package jicofo.
(Reading database ... 18927 files and directories currently installed.)
Preparing to unpack .../jicofo_1.0-121-1_amd64.deb ...
Unpacking jicofo (1.0-121-1) ...
Selecting previously unselected package authbind.
Preparing to unpack .../authbind_2.1.1_amd64.deb ...
Unpacking authbind (2.1.1) ...
Selecting previously unselected package javascript-common.
Preparing to unpack .../javascript-common_11_all.deb ...
Unpacking javascript-common (11) ...
Selecting previously unselected package libjs-jquery.
Preparing to unpack .../libjs-jquery_2.1.1-1_all.deb ...
Unpacking libjs-jquery (2.1.1-1) ...
Selecting previously unselected package libjs-jquery-ui.
Preparing to unpack .../libjs-jquery-ui_1.10.1+dfsg-1_all.deb ...
Unpacking libjs-jquery-ui (1.10.1+dfsg-1) ...
Selecting previously unselected package libjs-strophe.
Preparing to unpack .../libjs-strophe_1.1.3+dfsg-0.1_all.deb ...
Unpacking libjs-strophe (1.1.3+dfsg-0.1) ...
Selecting previously unselected package jitsi-meet-prosody.
Preparing to unpack .../jitsi-meet-prosody_1.0.603-1_all.deb ...
Unpacking jitsi-meet-prosody (1.0.603-1) ...
Selecting previously unselected package jitsi-meet.
Preparing to unpack .../jitsi-meet_1.0.603-1_all.deb ...
Unpacking jitsi-meet (1.0.603-1) ...
Processing triggers for systemd (215-17+deb8u1) ...
Setting up jicofo (1.0-121-1) ...
useradd: warning: the home directory already exists.
Not copying any file from skel directory into it.
invoke-rc.d: policy-rc.d denied execution of start.
Setting up authbind (2.1.1) ...
Setting up javascript-common (11) ...
Setting up libjs-jquery (2.1.1-1) ...
Setting up libjs-jquery-ui (1.10.1+dfsg-1) ...
Setting up libjs-strophe (1.1.3+dfsg-0.1) ...
Setting up jitsi-meet-prosody (1.0.603-1) ...
Generating a 4096 bit RSA private key
....................++
..................................................++
writing new private key to '/var/lib/prosody/localhost.key'
-----
invoke-rc.d: policy-rc.d denied execution of restart.
invoke-rc.d: policy-rc.d denied execution of restart.
invoke-rc.d: policy-rc.d denied execution of restart.
Setting up jitsi-meet (1.0.603-1) ...
/var/lib/dpkg/info/jitsi-meet.postinst: line 77: /usr/share/jitsi-videobridge/.sip-communicator/sip-communicator.properties: No such file or directory
dpkg: error processing package jitsi-meet (--configure):
 subprocess installed post-installation script returned error exit status 1
Processing triggers for systemd (215-17+deb8u1) ...
Errors were encountered while processing:
 jitsi-meet
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Any idea why that is?

**Comment by angelrr7702** [comment]
I have the same problem .. still waiting for fix for this error to be able to install this application .

**Comment by damencho** [comment]
I just tried on fresh installed ubuntu lts 14.04.3 and have no problem
installing jitsi-meet. Anything special for your installation?

**Comment by angelrr7702** [comment]
Did you install by 

echo 'deb http://download.jitsi.org/nightly/deb unstable/' >> /etc/apt/sources.list
wget -qO - https://download.jitsi.org/nightly/deb/unstable/archive.key | apt-key add -

apt-get update
apt-get -y install jitsi-meet

for me fail for Ubuntu 14.10 and Ubuntu 15.10
before it work fine at ubuntu 14.10 but for some reason .. it begin to fail some week ago : 
	2015-07-19  ==> installed fine 
        2015-08-13  ==> after this date fail to install from repository for me

This my last try error : 

Setting up jitsi-meet (1.0.626-1) ...
/var/lib/dpkg/info/jitsi-meet.postinst: line 77: /usr/share/jitsi-videobridge/.sip-communicator/sip-communicator.properties: No such file or directory
dpkg: error processing package jitsi-meet (--configure):

**Comment by damencho** [comment]
Yes I have tested the same, but till now have reports for 14.04. I will try on 14.10. Thanks.

**Comment by angelrr7702** [comment]
No idea if this will help you find the problem : 
before it work fine at ubuntu 14.10 but for some reason .. it begin to fail some week ago : 
2015-07-19 ==> installed fine    ==> Jitsi-meet   1.0.569-1
2015-08-13 ==> after this date fail to install from repository for me  ==> Jitsi-meet   1.0.606-1
2015-08-19 ==> still fail to install ==> Jitsi-meet   1.0.626-1

**Comment by damencho** [comment]
I created a clean installation of 14.10 and added the repository and did
just apt-get install jitsi-meet and it installed without a problem.
Are you upgrading or making clean install? If you are upgrading did you
made a modification like deleting the sip-communicator.properties file?

**Comment by angelrr7702** [comment]
I did  install inside a docker container .. in both way ... manual and by script . and both fail.
This before worked fine , ,, and the person that open the ticket have the same problem but from debian .

**Comment by damencho** [comment]
Well, I'm not familiar with docker and how the ubuntu 14.10 is different
there, but installing it on clean 14.10 is working fine so it must be
something docker specific that causes the problem.

**Comment by pailong** [comment]
Same problem under fresh install of debian wheezy : 

echo 'deb http://download.jitsi.org/nightly/deb unstable/' >> /etc/apt/sources.list
wget -qO - https://download.jitsi.org/nightly/deb/unstable/archive.key | apt-key add -
apt-get install jitsi-meet 

used to work a few days ago
Now fails.
New with the last install : nginx-nxasi-ui installs and requires a database

I have no idea how to fix.
Cheers

**Comment by damencho** [comment]
Hum, it is very strange I cannot reproduce any of the reports for this. I
just made a fresh install of wheezy and added the repository and did: apt-get
install jitsi-meet and everything installed without an issue. Any tips how
to reproduce this?

**Comment by damencho** [comment]
Found the problem where reconfiguring the package (changing the name of the service) can cause this problem. Not seen on new installation or simple upgrades. Fixed with 5a254ba.
